repondeur/tests/testapp.py

```diff
@@ -12,6 +12,23 @@ class TestResponse(BaseTestResponse):
     def parser(self):
         return HTMLParser(self.text)
 
+    @property
+    def first_h1(self) -> str:
+        return self.parser.css_first("h1").text()
+
+    def contains_amendement(self, amendement, is_gouvernemental=False) -> bool:
+        anchor = self.parser.css_first(f"#amendement-{amendement.num:06}")
+        if not anchor:
+            return False
+        assert (
+            str(amendement.num) in anchor.parent.css_first("header h2").text().strip()
+        )
+        if is_gouvernemental:
+            assert anchor.parent.attributes == {"class": "gouvernemental"}
+        else:
+            assert anchor.parent.attributes == {}
+        return True
+
```

Comment:
I found the asserts in the middle of a boolean method, itself used within an assert, a bit confusing.